doc/sphinx/language/core/basic.rst

```diff
@@ -523,24 +523,24 @@ they appear after a boldface label.  They are listed in the
 Locality attributes supported by :cmd:`Set` and :cmd:`Unset`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The :cmd:`Set` and :cmd:`Unset` commands support the :attr:`local`,
-:attr:`global` and :attr:`export` locality attributes:
-
-* no attribute: the original setting is *not* restored at the end of
-  the current module or section.
-* :attr:`local` (or alternatively, the ``Local`` prefix): the setting
+The :cmd:`Set` and :cmd:`Unset` commands support the mutually
+exclusive :attr:`local`, :attr:`export` and :attr:`global` locality
+attributes:
+
+* if no attribute is specified: the original value of the setting is
+  restored at the end of the current module but it is *not* restored
+  at the end of the current section.
+* :attr:`local` (or the ``Local`` prefix): the setting
   is applied within the current module or section.  The original value
   of the setting is restored at the end of the current module or
   section.
-* :attr:`export` (or alternatively, the ``Export`` prefix): similar to
-  :attr:`local`, the original value of the setting is restored at the
-  end of the current module or section.  In addition, if the value is
-  set in a module, then :cmd:`Import`\-ing the module sets the option
-  or flag.
-* :attr:`global` (or alternatively, the ``Global`` prefix): the
-  original setting is *not* restored at the end of the current module
-  or section.  In addition, if the value is set in a file, then
-  :cmd:`Require`\-ing the file sets the option.
+* :attr:`export` (or the ``Export`` prefix): same as no attribute, but
+  in addition, exported settings in :cmd:`Import`\ed modules override
+  the current value.
```

Comment:
```suggestion
* :attr:`export` (or the ``Export`` prefix): similar to no attribute, but
  settings exported from :cmd:`Import`\ed modules override
  the current value.
```

Or maybe "same as no attribute, except ...".  "In addition" is a weak way to convey the contradiction between the two clauses.  IMHO.